docs/create_new_docs.py

```diff
@@ -0,0 +1,8 @@
+from pathlib import Path
+import os
+import webbrowser
+
+path_makefile= Path().home() / "projects/manim-community/docs"
+os.system(f"cd {path_makefile} && make html")
+website = Path().home() / "projects/manim-community/docs/build/html/test.html"
+webbrowser.get('firefox').open_new_tab(f"{website}")
```

Comment:
Please see my comments on your other PR, which also adds the same file : https://github.com/ManimCommunity/manim/pull/291#discussion_r470960974

Reply:
I think you should delete this file from this branch, since it has been added to a different folder in the master branch.